implementations/nestjs/.gitignore

```diff
@@ -0,0 +1,398 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Note that this hefty `.gitignore` came from the NestJS quickstart repo. I decided to leave it as is.